subsys/esb/esb.c

```diff
@@ -1366,12 +1336,16 @@ static void on_radio_disabled_tx(void)
 	nrf_radio_packetptr_set(NRF_RADIO, rx_payload_buffer);
 	if (fast_switching) {
 		nrf_radio_task_trigger(NRF_RADIO, NRF_RADIO_TASK_RXEN);
+		while (!nrf_radio_event_check(NRF_RADIO, NRF_RADIO_EVENT_READY)) {
```

Comment:
Is this wait required here? The next possible event is reception of ACK or timeout, so no need to block here.

Reply:
Right, it is not needed.